package-lock.json

```diff
@@ -45,12 +45,14 @@
       }
     },
     "ajv": {
-      "version": "4.11.8",
-      "resolved": "https://registry.npmjs.org/ajv/-/ajv-4.11.8.tgz",
-      "integrity": "sha1-gv+wKynmYq5TvcIK8VlHcGc5xTY=",
+      "version": "5.2.2",
```

Comment:
There's a huge deal of unrelated changes and version bumps. Can you please revert all the changes to `package.json` and `package-lock.json` and run `npm i metasync metarhia-common` anew?